docs/planning/evidence/brief-gap-analysis-2026-08-22/round1-findings/C5-cost-model-selection.md

```
@@ -0,0 +1,862 @@
+
+  **The best number I can construct.** Fill-stage cost is measured, from
+  `docs/planning/vendor-comparison/runs/deepseek-v4-pro-2026-08-20/report.json` (DeepSeek V4 Pro,
+  `azure/us` pin, cap 131,072, `max_repairs=3`, reading-level passes 2, `stage1_gate=skipped`):
+
+  | Book | Band / nodes | Delivered leaf words | Output tok | **Measured cost** |
+  | --- | --- | ---: | ---: | ---: |
+  | 1 `the-last-cartage` | 16+ / 632 | 19,423 | 154,253 | **$1.0637** |
+  | 2 `the-quarry-signal` | 13-16 / 267 | 9,997 | 75,278 | **$0.5383** |
+  | 3 `the-tin-whistle-map` | 8-11 / 193 | 8,353 | 50,147 | **$0.3502** |
+  | 4 `the-last-blue-cup` (rerun) | 3-5 | 598 | 3,366 | **$0.0474** |
+  | 0 `the-last-cartage` | 16+ | 0  - | **unmetered** ("no provider call was metered for this book"), 397.96 s |
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 5; Actual: 4; Too few cells, row will be missing data

(MD056, table-column-count)